python_modules/libraries/dagster-dbt/dagster_dbt/core/resource.py

```diff
@@ -902,19 +871,25 @@ def get_dbt_resource_names_for_output_names(
     return [".".join(dbt_resource_props["fqn"]) for dbt_resource_props in dbt_resource_props_gen]
 
 
-def get_dbt_test_names_for_asset_checks(
+def get_dbt_test_names_for_check_keys(
```

Comment:
I think this name is wrong. We're actually extracting it from the check specs based on metadata right

Reply:
From the caller's perspective, they input a list of check keys and get dbt test names out, so seems reasonable to me

can be convinced otherwise

Reply:
Ah ok I think I missed the `check_keys` parameter being there on first pas